product_docs/docs/pgd/5/security/role-management.mdx

```diff
@@ -2,36 +2,36 @@
 title: Role management
 ---
 
-Users are global objects in a PostgreSQL instance. A `CREATE ROLE` command or its alias `CREATE USER` is replicated automatically if they're executed in a PGD replicated database. If a role or user is created in a non-PGD, un-replicated database, the role will only exist for that PostgreSQL instance. `GRANT ROLE` and `DROP ROLE` work in the same way, only replicating if applied to a PGD replicated database.
+Users are global objects in a PostgreSQL instance. A `CREATE ROLE` command or its alias `CREATE USER` is replicated automatically if it's executed in a PGD replicated database. If a role or user is created in a non-PGD, unreplicated database, the role exists only for that PostgreSQL instance. `GRANT ROLE` and `DROP ROLE` work the same way, replicating only if applied to a PGD-replicated database.
 
 !!! Note
-Remember that a user in Postgres terms is simply a role with login privileges.
+A user in Postgres terms is a role with login privileges.
```

Comment:
No. The emphasis on remembering and that this is a simple association was essential to the tone and positioning of this note.

Reply:
```suggestion
Remember that a user in Postgres terms is simply a role with login privileges.
```